### PR TITLE
Apply Settings to export

### PR DESCRIPTION
**Pull request by zkharit**
Add support for adding css to the export iframe

### CODE REVIEW

File: export/index.template.html [hunk 1]
```diff
@@ -671,6 +689,111 @@ <h2>Message log</h2>
...
+         * Function to validate and sanitize the styles object using the accepted map of style keys and values (as regular expressions).
+         * Any invalid style throws an error. Returns an object of valid styles.
+         * @param {Object} styles
+         * @return {Object}
+         */
+         function validateStyles(styles, element) {
+          const validStyles = {};
+
+          const cssValidationRegex = {
+            padding: "^(\\d+(px|em|%|rem) ?){1,4}$",
```
Comment (andrewkmin) [inline export/index.template.html:702]: Might make sense to put these valid, modifiable css properties in a doc somewhere

File: export/index.template.html [hunk 2]
```diff
@@ -671,6 +689,111 @@ <h2>Message log</h2>
...
+          if (settingsObj.styles) {
+            // Valid styles will be applied the "key-div" textarea HTML element.
+            const keyDivTextarea = document.getElementById("key-div");
+            if (!keyDivTextarea) {
+              throw new Error(
+                "no key-div HTML element found to apply settings to."
+              );
+            }
+
+            // Validate, sanitize, and apply the styles to the "key-div" textarea.
```
Comment (andrewkmin) [inline export/index.template.html:785]: The `key-div` element is just a simple div, which later gets its `innerText` set

File: export/index.template.html [hunk 3]
```diff
@@ -709,6 +836,11 @@ <h2>Message log</h2>
       document.addEventListener(
         "DOMContentLoaded",
         async () => {
+          // If styles are saved in local storage, sanitize and apply them.
```
Comment (andrewkmin) [inline export/index.template.html:839]: You may have already done this, and nothing indicates the contrary, but would be nice to do a sanity check to make sure these loaded settings can be overridden if the user calls `applySettings` again

**Review by andrewkmin — APPROVED** [review]
non blocking comments.

let's run prettier here `npm run prettier:write`

File: export/index.template.html (hunk 2 above)
Comment (zkharit) [reply export/index.template.html]: changed this

File: export/index.template.html (hunk 3 above)
Comment (zkharit) [reply export/index.template.html:842]: Yes this works as exepected!

File: export/index.template.html (hunk 1 above)
Comment (zkharit) [reply export/index.template.html:705]: good idea, can add something to the UI customization page

**Pull request merged**